mars/services/cluster/api/core.py

```diff
@@ -188,3 +188,19 @@ async def fetch_node_log(self, size: int, address: str, offset: int) -> str:
         -------
 
         """
+
+    @abstractmethod
+    async def save_yaml(self, obj: Dict, save_path: str, address: str) -> None:
```

Comment:
The argument `address` is not necessary since the data is always saved on the localhost.

Reply:
`save_yaml` is a little bit too general. How about `save_task_info` or `append_task_info`?